tests/page-objects/users/add-user-modal.wdio.page.js

```diff
@@ -0,0 +1,45 @@
+const getUsernameField = () => $('#edit-username');
```

Comment:
I think we already have a utility for the admin users modal here: https://github.com/medic/cht-core/blob/master/tests/page-objects/admin/user.wdio.page.js Can we reuse that instead of adding a new one?

Reply:
removed this file and used existing one